src/WorkflowManager/Controllers/WorkflowsController.cs

```diff
@@ -118,11 +120,14 @@ public async Task<IActionResult> GetAsync([FromRoute] string id)
     [HttpPost]
     public async Task<IActionResult> CreateAsync([FromBody] Workflow workflow)
     {
-        if (!workflow.IsValid(out var validationErrors))
+        var validator = new WorkflowValidator();
```

Comment:
Can this not be made static?

Reply:
I've updated this to be static instance now